applications/FluidDynamicsApplication/python_scripts/adjoint_monolithic_solver.py

```diff
@@ -90,10 +152,19 @@ def AddVariables(self):
         KratosMultiphysics.Logger.PrintInfo(self.__class__.__name__, "Adjoint fluid solver variables added correctly.")
 
     def Initialize(self):
+        # If the solver requires an instance of the stabilized formulation class, set the process info variables
+        if hasattr(self, 'formulation'):
+            self.formulation.SetProcessInfo(self.GetComputingModelPart())
+
         # Construct and set the solution strategy
         solution_strategy = self._GetSolutionStrategy()
         solution_strategy.SetEchoLevel(self.settings["echo_level"].GetInt())
 
+        # clear RELAXED_ACCELERATION if it is a steady adjoint problem
+        scheme_type = self.settings["scheme_settings"]["scheme_type"].GetString()
+        if (scheme_type == "steady"):
+            KratosMultiphysics.VariableUtils().SetNonHistoricalVariableToZero(KratosMultiphysics.RELAXED_ACCELERATION, self.main_model_part.Nodes)
```

Comment:
Above you add `RELAXED_ACCELERATION` to the historical database. However in here you initialize it in the non-historical. I think you want to keep either one or the other.

Reply:
removed from historical container. 👍